README.md

```diff
@@ -154,6 +154,14 @@ You should see the new version number on startup, for example: `[ TASKLOG ]  Wel
 
 You're done! And don't forget to have a look at the remaining functions of the `setup.sh` script with `./setup.sh help`.
 
+#### Build Your Own Docker Image
+You'll need to retrieve the git submodules prior to building your own Docker image. From within your copy of the git repo, run:
```

Comment:
Title is a bit vague.

- Building own custom Dockerfile or the one in git?
- Building own Docker image could also extend ours as a base image

Perhaps "Building the project Dockerfile from git" or similar? (maybe source instead of git)

In my unpublished docs it was for devs/contributors to build as part of testing their PR locally, but for users I think they can have different intentions for their own docker image.

Reply:
IMO the title is fine and I don't think, that there will be misunderstandings. Building docker images also presupposes a basic knowledge about it.

BTW, to make it even simpler: Instead of "run `git submodules..` in your previous cloned repo" etc., maybe we should just advice to `git clone --recurse-submodules https://github.com/docker-mailserver/docker-mailserver`.

1. Clone repo (recursively) --> `git clone..`
2. Build image --> `docker build -t  my-DMS-build .`